float64[] y_actual
float64[] y_predicted
string action
---
float64 err
